ice.c

```diff
@@ -4016,6 +4031,23 @@ void janus_ice_relay_data(janus_ice_handle *handle, char *buf, int len) {
 }
 #endif
 
+void janus_ice_relay_sctp(janus_ice_handle *handle, char *buffer, int length) {
+#ifdef HAVE_SCTP
+	if(!handle || handle->queued_packets == NULL || buffer == NULL || length < 1)
+		return;
+	/* Queue this packet */
+	janus_ice_queued_packet *pkt = g_malloc(sizeof(janus_ice_queued_packet));
+	pkt->data = g_malloc(length);
```

Comment:
I'm not sure there's a need to allocate anything: IIRC the buffer you get from the SCTP stack is already allocated, meaning we can just "steal" it,

    pkt->data = buffer;

On the other end, this is what we were doing in `janus_sctp_message_create` too, so better keep it as you did it for now and possibly revisit it later on.

Reply:
I would do that in a separate PR if necessary.